libpince/typedefs.py

```diff
@@ -328,20 +328,24 @@ class SCAN_TYPE:
     CHANGED = 8
     UNCHANGED = 9
     UNKNOWN = 10
+    NOT = 11
 
     @staticmethod
-    def get_list(scan_mode):
+    def get_list(scan_mode, value_type):
+        list = []
```

Comment:
Just FYI, this line is not needed since there is no block scope in Python. Anything declared in the if and else below belong to the whole function.

No need to make another commit to change this though, just have it in mind for the future.

Reply:
It's aight, I can delete the redundant code later on